monyhar/PRESUBMIT.py

```diff
@@ -3916,7 +3916,7 @@ def CheckCorrectProductNameInMessages(input_api, output_api):
     "correct_name": "Chrome",
     "incorrect_name": "Chromium",
   }, {
-    "filename_postfix": "chromium_strings.grd",
+    "filename_postfix": "monyhar_strings.grd",
     "correct_name": "Chromium",
```

Comment:
```suggestion
     "correct_name": "Monyhar",
```
I think you didn't replace capitalized words?

Reply:
此处是灵魂